src/components/Banner.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+//import { ArrowRightCircle } from "react-bootstrap-icons";
+import headerImg from "../assets/img/yoshi 3.png";
+import music from '../assets/music.mp3';
+import { useEffect, useRef } from "react";
+import { BsFillPauseFill, BsFillPlayFill } from "react-icons/bs";
+
+export const Banner = () => { 
+  const [value, setValue] = useState(1);
```

Comment:
is there a reason why we are using number here instead of a boolean?

Reply:
fixed